### DISCUSSION TITLE
New Release: Espressif IDE 3.1.0 · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#177] [General]
# New Release: Espressif IDE 3.1.0 · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/espressif-ide/espressif-ide-3-1-0-release/

**Comment by Lsitar** [comment]
Hello, I recently updated Espressif IDE from 2.7 to 3.1 and there are new features I'd like to disable:
- in functions usage, displaying of function parameter names before parameter values,
- automatic adding "missing" includes in background
Can You tell a path to the setting? I couldn't find it.

**Reply by kolipakakondal** [reply]
Hi @Lsitar To disable the inlay hints, please check this https://github.com/espressif/idf-eclipse-plugin/tree/56fcb66aae24ce04a48a91e50774a46541ae0dd6?tab=readme-ov-file#inlay-hints

**Reply by kolipakakondal** [reply]
Auto import already works with the LSP Editor in Espressif-IDE 3.1.0 release. Attaching the screenshare for your reference.

https://github.com/user-attachments/assets/d4628a17-5bf9-4307-a8a5-eda69141be33

**Comment by zggcd157** [comment]
目前最新的espressif ide只支持5.3.1吗？
